qiskit/circuit/library/arithmetic/adders/cdkm_ripple_carry_adder.py

```diff
@@ -75,6 +74,21 @@ class CDKMRippleCarryAdder(Adder):
     It has one less qubit than the full-adder since it doesn't have the carry-out, but uses
     a helper qubit instead of the carry-in, so it only has one less qubit, not two.
 
+    .. seealso::
+
```

Comment:
I did not know about ".. seealso::" before, but apparently we already use it quite a bit in Qiskit.